### PR TITLE
Remove mention of simulated results in CHSH tutorial

### LINKED ISSUE #1443: Update the CHSH tutorial
### URL to the relevant documentation

https://learning.quantum.ibm.com/tutorial/chsh-inequality

### Select all that apply

- [ ] typo
- [ ] code bug
- [X] out-of-date content
- [ ] broken link
- [ ] other

### Describe the fix.

Issues identified by Paul N:

The CHSH tutorial is tagged "scheduling", but there is no scheduling involved.

Also the text below the final figure references simulation results that no longer exist in the tutorial

### PR DESCRIPTION
**Pull request by beckykd**
First part of #1443 

Still need to figure out how to change the tags on the platform.  

- [ ]  The CHSH tutorial is tagged "scheduling", but there is no scheduling involved.

### CODE REVIEW

**Review by Eric-Arellano — APPROVED** [review]
Thanks! For my own curiosity, why remove the colors?

**Comment by beckykd** [comment]
> Thanks! For my own curiosity, why remove the colors?

Because they're hard to see and I've seen several comments about people being confused.  Since one is just dot, and one is dot-dash, I think that's good enough to identify the bounds.

**Comment by Eric-Arellano** [comment]
> Does that seem reasonable?

Yeah. Looking at it, even the dashes are very hard to distinguish. Thoughts on calling it the "inner lines" and "outer lines", or something?

Screenshot 2024-05-28 at 4 36 54 PM

File: tutorials/chsh-inequality/chsh-inequality.ipynb [hunk 1]
```diff
@@ -432,7 +432,7 @@
    "id": "63e63853",
    "metadata": {},
    "source": [
-    "In the figure, the red dashed lines delimit the classical bounds ($\\pm 2$) and the dash-dotted blue lines delimit the quantum bounds ($\\pm 2\\sqrt{2}$). You can see that the experimental results resemble the general trend of the simulated results and there are regions where the CHSH witness quantities exceeds the classical bounds. Congratulations! You have successfully demonstrated the violation of CHSH inequality in a real quantum system!"
+    "In the figure, the dashed lines delimit the classical bounds ($\\pm 2$) and the dash-dotted lines delimit the quantum bounds ($\\pm 2\\sqrt{2}$). You can see that there are regions where the CHSH witness quantities exceeds the classical bounds. Congratulations! You have successfully demonstrated the violation of CHSH inequality in a real quantum system!"
```
Comment (beckykd) [inline tutorials/chsh-inequality/chsh-inequality.ipynb:435]:
```suggestion
    "In the figure, lines and the gray areas in between delimit the bounds; the outer-most (dash-dot) lines delimit the quantum-bounds ($\\pm 2$) whereas the inner (dashed) lines delimit the classical bounds ($\\pm 2\\sqrt{2}$). You can see that there are regions where the CHSH witness quantities exceeds the classical bounds. Congratulations! You have successfully demonstrated the violation of CHSH inequality in a real quantum system!"
```

**Comment by beckykd** [comment]
The tags are being taken care of by @frankharkins

**Review by Eric-Arellano — APPROVED** [review]

**Pull request merged**